object/types.proto

```diff
@@ -160,3 +160,21 @@ message Object {
   // Payload bytes.
   bytes payload = 4 [json_name = "payload"];
 }
+
+// Meta information of split hierarchy for object assembly. With right child
+// one can traverse linked list back to left or first child to assembly
+// original object. With link object one can assembly object straight
+// away from the object parts.
+message SplitInfo {
+  // 16 byte UUID used to identify the split object hierarchy parts.
+  bytes split_id = 1;
+
+  // Identifier of most right child in split hierarchy parts. It contains
+  // split header with original object header.
+  neo.fs.v2.refs.ObjectID right_child_id = 2;
```

Comment:
Suggest to use 'last_part' or something similar

Reply:
Fixed.